content/en/training/_index.html

```diff
@@ -63,43 +64,45 @@ <h5>
   </div>
 </section>
 
-<div class="padded lighter-gray-bg">
-  <div class="main-section two-thirds-centered">
+<div class="padded lighter-gray-bg-banner">
```

Comment:
**Part 2:** 
Use `<section class="padded lighter-gray-bg-banner">` not `<div>`. The CSS class which center things properly in a container are accessed in the stylesheet via `section.div`, ergo it doesn't work :)

Next, you need to have a second div (annoying, I know) for `<div class="main-section">` here. This allows the gray background to be full-bleed while the content is constrained to a particular width. 

Putting it all together, lines 67-69 should look like this: 

```
<section class="padded lighter-gray-bg-banner">
  <div class="main-section">
    <div class="half">
       //Learn w/Linux Foundation content here
    </div> 
    <div class="half-image>
      <img src="...">
    </div>
  </div>
</section>
```